distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerImpl.java

```diff
@@ -263,16 +289,36 @@ private FeaturesService getFeaturesService() throws InterruptedException {
                 }
                 Thread.sleep(1000);
             }
+
         }
 
         return featuresService;
     }
 
```

Comment:
It looks like the Jenkins failed because of a jacoco constraint in /Users/aaronhoffer/repos/connexta/ddf/ddf/platform/admin/core/admin-core-api/pom.xml

 > on project admin-core-api: Coverage checks have not been met.